kuryr_kubernetes/controller/drivers/lbaasv2.py

```diff
@@ -839,8 +845,12 @@ def _find_loadbalancer(self, loadbalancer):
             loadbalancer.port_id = self._get_vip_port(loadbalancer).get("id")
             loadbalancer.provider = os_lb.provider
             if os_lb.provisioning_status == 'ERROR':
```

Comment:
What about the LB in status 'PENDING_DELETE', do we need to check for that too?

Reply:
It's not possible to release LBs with status PENDING_*, this is an immutable state